lego/apps/events/fixtures/test_events.yaml

```diff
@@ -264,7 +264,7 @@
     name: Abakusmember
     capacity: 2
     event: 10
-    counter: 2
+    counter: 1
```

Comment:
This was incorrect, since the test making sure that the task for ensuring the counter is correct of course did not work. Now that the task is fixed, the tests now failed on this event of course.

This was prob just copy-pasted, so should not have any negative effects.